src/pages/PullRequestPage/subroute/IndirectChangesTab/IndirectChangedFiles/hooks/useIndirectChangedFilesTable.spec.js

```diff
@@ -1,122 +1,135 @@
 
 import { useIndirectChangedFilesTable } from './useIndirectChangedFilesTable'
 
-jest.mock('react-router-dom', () => ({
-  ...jest.requireActual('react-router-dom'),
-  useParams: jest.fn(() => {}),
-}))
-jest.mock('services/pull')
-
-const mockImpactedFiles = [
-  {
-    missesCount: 0,
-    isCriticalFile: true,
-    fileName: 'mafs.js',
-    headName: 'flag1/mafs.js',
-    baseCoverage: {
-      percentCovered: 45.38,
-    },
-    headCoverage: {
-      percentCovered: 90.23,
-    },
-    patchCoverage: {
-      percentCovered: 27.43,
-    },
-  },
-]
-
-let mockPull = {
-  data: {
-    pull: {
-      pullId: 14,
-      head: {
-        state: 'PROCESSED',
-      },
-      compareWithBase: {
-        __typename: 'Comparison',
-        patchTotals: {
-          percentCovered: 92.12,
-        },
-        headTotals: {
-          percentCovered: 74.2,
+const mockData = {
+  owner: {
+    repository: {
+      pull: {
+        pullId: 14,
+        head: {
+          state: 'PROCESSED',
         },
-        baseTotals: {
-          percentCovered: 27.35,
-        },
-        changeCoverage: 38.94,
-        impactedFiles: {
-          __typename: ImpactedFilesReturnType.IMPACTED_FILES,
-          results: mockImpactedFiles,
+        compareWithBase: {
+          __typename: 'Comparison',
+          patchTotals: {
+            percentCovered: 92.12,
+          },
+          headTotals: {
+            percentCovered: 74.2,
+          },
+          baseTotals: {
+            percentCovered: 27.35,
+          },
+          changeCoverage: 38.94,
+          impactedFiles: {
+            __typename: ImpactedFilesReturnType.IMPACTED_FILES,
+            results: [
+              {
+                missesCount: 0,
+                isCriticalFile: true,
+                fileName: 'mafs.js',
+                headName: 'flag1/mafs.js',
+                baseCoverage: {
+                  percentCovered: 45.38,
+                },
+                headCoverage: {
+                  percentCovered: 90.23,
+                },
+                patchCoverage: {
+                  percentCovered: 27.43,
+                },
+              },
+            ],
+          },
         },
       },
     },
   },
 }
 
+const queryClient = new QueryClient({
+  defaultOptions: { queries: { retry: false } },
+})
+const server = setupServer()
+
+const wrapper =
+  (initialEntries = '/gh/test-org/test-repo/pull/5?flags=a,b') =>
+  ({ children }) =>
+    (
+      <QueryClientProvider client={queryClient}>
+        <MemoryRouter initialEntries={[initialEntries]}>
+          <Route path="/:provider/:owner/:repo/pull/:pullId">{children}</Route>
+        </MemoryRouter>
+      </QueryClientProvider>
+    )
+
+beforeAll(() => {
+  server.listen()
+  console.error = () => {}
+})
+afterEach(() => {
+  queryClient.clear()
+  server.resetHandlers()
+})
+afterAll(() => server.close())
+
 describe('useRepoContentsTable', () => {
```

Comment:
nit: this isn't the function that's being tested here, seems to be in a couple of other places in here 👀

Reply:
Might be an old typo that slipped though in previous PRs, I could give it the once over. Do you have the other files you noticed?

Reply:
It's all in the same file